src/templates/resources/header-docs-data.json

```diff
@@ -5,6 +5,7 @@
   "logoutUrl": "#",
   "helpUrl": "#",
   "logoUrl": "/assets/img/logos/je-logo.v1.svg",
+  "logoUrlMenulog": "/assets/img/logos/je-logo.v2.menulog.svg",
```

Comment:
Wonder if we should have more of a convention for this naming – maybe `logoUrl-menulog` so that it's clear, or `menulog-logoUrl` if we wanted them all grouped together?

What do you reckon /cc @DamianMullins @simonsandersje @kevinrodrigues ?

Reply:
You should be able to change the logo filename to `je-logo.v2.au.svg` and the asset helper will handle passing the correct URL, right? That way you don't need the additional property.

Reply:
Also, I wouldn't worry about the property to this file as it is purely here for the Fozzie docs site which [assumes it is the Canadian tenant](https://github.com/justeat/f-header/blob/cb96f83b9918ac20ff57046702ad4abb330852fa/src/templates/resources/header-docs-data.json#L2).

Reply:
Might be worth changing the fie to be `je-logo.v1.au.svg` though instead of Menulog 👍

Reply:
@DamianMullins OrderWeb is using it. Please see https://github.je-labs.com/CPD-Order/OrderWeb/pull/534/files/9c9ce33b3cc2f32ea9fdcea09b3cde239951d49b#diff-17768ef6f648c64c2771156e2f8bda00
Could you suggest an alternative way?

@ashleynolan Do you think using just au is ok here, since menulog branding is in NZ as well.

Reply:
Ah, you'll need an `je-logo.v1.en.au.svg` and a `je-logo.v1.en.nz.svg` I think.

Reply:
OrderWeb shouldn't need to use the property either for the same reason; the asset helper will resolve to the correct image file depending on the current tenant.

Reply:
@bulutcy Is OrderWeb using values from this file?  If so, I think this file is only intended for docs use.  That's correct right @DamianMullins ?

Reply:
thanks, will fix that. I wasn't aware of that feature.